src/Views/Admin/AddTeamComponents/ManageMmbrs.jsx

```diff
@@ -103,7 +121,17 @@ const ManageMmbrs = () => {
             </div>
           </div>
         )}
-        <div className="adt_addplayerwrap"></div>
+        {/* MAP  MEMBERCARDS INSIDE THIS DIV CONTAINER  */}
+        <div className="adt_addplayerwrap">
+          <MembersCard
+            memberName="TAKERU SATOH"
+            memberProfileType="PLAYER"
+            memberAddress="TOKYO, JAPAN"
+            discordLink="https://discord.com"
+            fbLink="https://fb.com"
+            twitchLink="https://twitch.com"
+          />
```

Comment:
Please create an array of objects and use map please. To avoid confusion when implementing backend functions.

Reply:
![q6](https://github.com/Hitoru141/EsportsWebsite/assets/36213437/b06ba598-b279-4b00-8d9f-85854cb873d6)
![q6](https://github.com/Hitoru141/EsportsWebsite/assets/36213437/02f60998-88cc-42fc-9a24-cb5eefcaa4c9)